data/json/obsoletion/overmap_special/specials.json

```
@@ -337,5 +337,208 @@
     "city_distance": [ 20, -1 ],
     "occurrences": [ 0, 0 ],
     "flags": [ "UNIQUE" ]
+  },
+  {
+    "type": "overmap_special",
+    "id": "Farm",
+    "overmaps": [
+      { "point": [ 0, 0, 0 ], "overmap": "farm_3_north" },
+      { "point": [ 1, 0, 0 ], "overmap": "farm_2_north" },
+      { "point": [ 2, 0, 0 ], "overmap": "farm_1_north" },
+      { "point": [ 0, 1, 0 ], "overmap": "farm_6_north" },
+      { "point": [ 1, 1, 0 ], "overmap": "farm_5_north" },
+      { "point": [ 2, 1, 0 ], "overmap": "farm_4_north" },
+      { "point": [ 0, 2, 0 ], "overmap": "farm_9_north" },
+      { "point": [ 1, 2, 0 ], "overmap": "farm_8_north" },
+      { "point": [ 2, 2, 0 ], "overmap": "farm_7_north" },
+      { "point": [ 0, 0, 1 ], "overmap": "farm_3_roof_north" },
+      { "point": [ 1, 0, 1 ], "overmap": "farm_2_roof_north" }
+    ],
+    "connections": [ { "point": [ 1, -1, 0 ], "terrain": "road", "connection": "local_road", "from": [ 1, 0, 0 ] } ],
+    "locations": [ "field" ],
+    "city_distance": [ 5, 40 ],
+    "city_sizes": [ 1, -1 ],
+    "occurrences": [ 0, 0 ],
```

```suggestion
    "occurrences": [ 0, 2 ],
```
0, 2 or three would be my suggestion